aiida/backends/tests/export_and_import.py

```diff
@@ -2142,3 +2145,210 @@ def test_that_input_code_is_exported_correctly(self):
                               .format(qb.count()))
         finally:
             shutil.rmtree(tmp_folder, ignore_errors=True)
+
+    @skip("")
+    def test_that_input_code_is_exported_correctly(self):
+        """
+        This test checks that when a calculation is exported then the
+        corresponding code is also exported. It also checks that the links
+        are also in place after the import.
+        """
+        import os, shutil, tempfile
+
+        from aiida.orm.utils import load_node
+        from aiida.orm.importexport import export
+        from aiida.common.links import LinkType
+        from aiida.orm.calculation.job import JobCalculation
+        from aiida.orm.code import Code
+        from aiida.orm.querybuilder import QueryBuilder
+
+        tmp_folder = tempfile.mkdtemp()
+
+        try:
+            code_label = 'test_code1'
+
+            code = Code()
+            code.set_remote_computer_exec((self.computer, '/bin/true'))
+            code.label = code_label
+            code.store()
+
+            code_uuid = code.uuid
+
+            jc = JobCalculation()
```

Comment:
`JobCalculation` no longer exists, use `CalculationNode` or `CalcJobNode` depending on what you want to test